Frontend/ui-customer/index.js

```diff
@@ -122,37 +122,22 @@ window.onMerchantEvent = function (_event, merchantPayload) {
   console.log(merchantPayload);
   const clientPaylod = JSON.parse(merchantPayload).payload;
   if (_event == "initiate") {
-    const payload = {
-      event: "initiate_result"
-      , service: "in.juspay.becknui"
-      , payload: { status: "SUCCESS" }
-      , error: false
-      , errorMessage: ""
-      , errorCode: ""
-    }
     let clientId = clientPaylod.clientId;
     if (clientId.includes("_ios"))
     {
       clientId = clientId.replace("_ios","");
     }
-    if (clientId == "open-kochi" || clientId == "yatriconsumer") {
-      window.merchantID = "YATRI"
-    } else if (clientId == "jatrisaathi" || clientId == "jatrisaathiconsumer"){
-      window.merchantID = "YATRISATHI"
-    } else if (clientId.includes("mobility")) {
-      let merchant = clientId.replace("mobility","")
-      merchant = merchant.replace("consumer","")
-      merchant = merchant.toUpperCase();
-      window.merchantID = "MOBILITY_" + merchant.charAt(0) + merchant.charAt(merchant.length - 1);
-    } else if (clientId.includes("consumer")) {
+    if (!clientId.startsWith("mobility") && clientId.includes("consumer")) {
```

Comment:
Can you explain what all is updated here

Reply:
It is to handle the app case and mobility case

Reply:
yeah, but how and what is changed

Reply:
if statement I understood, I'm asking about what is changed inside of `ifelse` block